`DimeNet++`

thegodone opened the pull request:
This is the code and example on MoleculeNet ESOL loader and Feature generator.

Comments part are here to related the fix in Distance / RBF Envelopp / ...  

this is to fix issue #2403

thegodone committed:
Merge pull request #1 from rusty1s/master

refresh last PyG

thegodone committed:
DimenetPP core and example

thegodone committed:
speed up performance 30%

rusty1s commented:
Hi @thegodone. Thanks for this PR. Is it possible for you to implement a `DimeNet++` model directly in PyG (at best by re-using some functionality from the `DimeNet` model?

thegodone commented:
I can but it means that I have to add 4,5 modifications in the current PyG Dimenet part. Should I set a "fix" parameter of a "2.0" version parameter in those functions/classes ?

rusty1s commented:
Can we make that an internal flag, i.e., `self._pretrained_version`, which just get sets to `True` in case we are calling `from_pretrained`?

thegodone commented:
Yes we can do that, it makes sense. For existing Dimenet pretrained model is it per target or a multitask model stored in TF and reload in PyG ?

rusty1s commented:
It is per target.

thegodone committed:
implementing the fix version of Dimenet++

those modifications allows to make Dimenet++ bug free and still compatible with original Dimenet model

thegodone commented:
I implement the code in the Dimenet PyG code and Add a fix bool parameter
